package com.kollaboralabs.auth.exception;

public class LoginAlreadyInUseException extends RuntimeException {
}